bot/exts/christmas/advent_of_code/_helpers.py

```diff
@@ -393,8 +398,8 @@ def is_in_advent() -> bool:
     return datetime.datetime.now(EST).day in range(1, 25) and datetime.datetime.now(EST).month == 12
 
 
-def time_left_to_aoc_midnight() -> Tuple[datetime.datetime, datetime.timedelta]:
-    """Calculates the amount of time left until midnight in UTC-5 (Advent of Code maintainer timezone)."""
+def time_left_to_est_midnight() -> Tuple[datetime.datetime, datetime.timedelta]:
+    """Calculate the amount of time left until midnight EST/UTC-5."""
```

Comment:
This isn't really important, but if you're generalizing this function, you might as well accept a timezone parameter, instead of only returning EST (you'd just have to move the EST constant into the function and pass it the timezone).

Reply:
This function is meant to be used in AoC cog and utility, so having it accept here timezone doesn't have point. When we make this helper function for all cogs, then this may have reason.

Reply:
I don't think it's necessary at the moment to factor it out at the moment.